scripts/byobu_startup.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import subprocess
+import sys
+import time
+
+from huntsman.pocs.utils import load_config
+
+WINDOWS = ["main-control",
+           "shutter-and-weather",
+           "camera-servers",
+           "camera-logs",
+           "dome-control",
+           "data managment"]
+
+POCS_STARTUP = ["from pocs.mount import create_mount_from_config",
+                "from pocs.scheduler import create_scheduler_from_config",
+                "from huntsman.pocs.camera import create_cameras_from_config",
+                "from huntsman.pocs.observatory import HuntsmanObservatory",
+                "from pocs.core import POCS",
+                "from huntsman.pocs.utils import load_config",
+                "config = load_config",
+                "cameras = create_cameras_from_config",
+                "mount = create_mount_from_config(config)",
+                "mount.initialize()",
+                "scheduler = create_scheduler_from_config(config)",
+                "observatory = HuntsmanObservatory(cameras=cameras, "
+                "mount=mount, scheduler=scheduler, with_autoguider=True,"
+                " take_flats=True",
+                "pocs = POCS(observatory, simulator=['power','weather']",
+                "#pocs.initialize()",
+                "#pocs.run()"]
```

Comment:
This kind of gets lost in the commands.

```suggestion
                # Uncomment the following lines for automatic running
                "#pocs.initialize()",
                "#pocs.run()"]
```